redis/3.2/README.md

```diff
@@ -1,7 +1,7 @@
 # REDIS 3.2
 
 ```Dockerfile
-FROM quay.io/continuouspipe/redis:3.2
+FROM quay.io/continuouspipe/redis:3.2_v1
```

Comment:
Yeah @kierenevans and I just discovered this on the Magento varnish image to. New commit coming just shortly :)